catalog/app/components/Assistant/UI/Chat/Chat.tsx

```diff
@@ -2,6 +2,7 @@ import cx from 'classnames'
 import * as Eff from 'effect'
 import * as React from 'react'
 import * as M from '@material-ui/core'
+import * as Icons from '@material-ui/icons'
```

Comment:
Please, use destructuring here and direct names for icons you are exporting

`import { CheckCircleOutline as IconCheckCircleOutline, ErrorOutline as IconErrorOutline } from '@material-ui/icons'`

Reply:
i don't think it's a good idea -- it adds verbosity and repetition for no real benefit. bundle size effect seems to be the same, since it's getting tree-shaked ("Everything is in check (+97.16KB, -96.62KB, +0.0%)").
to determine the exact icons used throughout the codebase you can just grep for `Icons.`
i believe qualified (namespaced) imports are better in general, because the qualifiers (namespaces) add helpful context about the variables, and reduce maintenance burden (you don't have to add/remove individual imports).

Reply:
I'm ok with both formats. If you're going to use wildcard imports, then please change existing imports of icons

Reply:
actually, mining the codebase for all the icons used (if that's actually something that will be done)
is easier with namespace imports, because it will give you the exact icon names in a single step (e.g. `... Icons.Error ...`), while doing so for individual import requires two steps: grep for `'@material-ui/icons'` import, and then see all the individual imports (because in some cases they will be split across multiple lines).

so, using individual imports seems like a lose-lose scenario (more effort, less value).

Reply:
It makes more sense to do this in a standalone PR

Reply:
When are you planning to make a new PR?

Reply:
Good